static/js/util/google_analytics.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+/* global SETTINGS:false */
+import ga from "react-ga";
+import R from "ramda";
+
+const makeGAEvent = (category, action, label, value) => ({
+  category: category,
+  action:   action,
+  label:    label,
+  value:    value
+});
+
+const removeNilValue = R.when(
+  R.compose(R.isNil, R.prop("value")),
+  R.dissoc("value")
+);
+
+const formatGAEvent = R.compose(removeNilValue, makeGAEvent);
+
+export const sendGAEvent = (
+  category: string,
+  action: string,
+  label: string,
+  value?: number
+) => {
+  ga.event(formatGAEvent(category, action, label, value));
```

Comment:
Can you add a `Math.round(value)` here and also a check that `value` is a `number`?

Reply:
Updated (`Math.round` at `makeGAEvent`; new `isValidNumber` function).